src/app/pages/reader/searcher/searcher.component.ts

```diff
@@ -38,4 +40,7 @@ export class SearcherComponent {
       this.results = [];
     }
   }
+  viewUserProfile(userId: number): void {
+    this.router.navigate(['/user/profile', userId]);
```

Comment:
Por qué se usa this,router,navigate?

Reply:
Para ir a la página del perfil del usuario